src/main/java/org/fogbowcloud/manager/core/models/orders/instances/OrderInstance.java

```diff
@@ -1,14 +1,56 @@
 package org.fogbowcloud.manager.core.models.orders.instances;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "tb_order_instance")
 public class OrderInstance {
 
+	@Id
+	@Column(name = "id", nullable = false, unique = true)
     private String id;
 
+	public OrderInstance() {
+
+	}
+
+	public OrderInstance(String id) {
+		this.id = id;
+	}
+
     public String getId() {
         return id;
     }
 
     public void setId(String id) {
         this.id = id;
     }
+
+	@Override
+	public int hashCode() {
```

Comment:
hashCode and equals methods could be implemented at the Order superclass for code reuse.